src/main/resources/generator/ios-moe/xcode/ios-moe.xcodeproj/project.pbxproj

```diff
@@ -1,595 +1,595 @@
-		581773911E37A0F0004E28A9 /* Default-375w-667h@2x.png in Resources */ = {isa = PBXBuildFile; fileRef = 581773851E37A0F0004E28A9 /* Default-375w-667h@2x.png */; };
-		581773921E37A0F0004E28A9 /* Default-414w-736h@3x.png in Resources */ = {isa = PBXBuildFile; fileRef = 581773861E37A0F0004E28A9 /* Default-414w-736h@3x.png */; };
-		581773931E37A0F0004E28A9 /* Default-568h@2x.png in Resources */ = {isa = PBXBuildFile; fileRef = 581773871E37A0F0004E28A9 /* Default-568h@2x.png */; };
-		581773941E37A0F0004E28A9 /* Default-1024w-1366h@2x~ipad.png in Resources */ = {isa = PBXBuildFile; fileRef = 581773881E37A0F0004E28A9 /* Default-1024w-1366h@2x~ipad.png */; };
-		581773951E37A0F0004E28A9 /* Default.png in Resources */ = {isa = PBXBuildFile; fileRef = 581773891E37A0F0004E28A9 /* Default.png */; };
-		581773961E37A0F0004E28A9 /* Default@2x.png in Resources */ = {isa = PBXBuildFile; fileRef = 5817738A1E37A0F0004E28A9 /* Default@2x.png */; };
-		581773971E37A0F0004E28A9 /* Default@2x~ipad.png in Resources */ = {isa = PBXBuildFile; fileRef = 5817738B1E37A0F0004E28A9 /* Default@2x~ipad.png */; };
-		581773981E37A0F0004E28A9 /* Default~ipad.png in Resources */ = {isa = PBXBuildFile; fileRef = 5817738C1E37A0F0004E28A9 /* Default~ipad.png */; };
-		581773A61E37A2DB004E28A9 /* AudioToolbox.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5817739F1E37A2DB004E28A9 /* AudioToolbox.framework */; };
-		581773A71E37A2DB004E28A9 /* AVFoundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 581773A01E37A2DB004E28A9 /* AVFoundation.framework */; };
-		581773A81E37A2DB004E28A9 /* CoreGraphics.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 581773A11E37A2DB004E28A9 /* CoreGraphics.framework */; };
-		581773A91E37A2DB004E28A9 /* CoreMotion.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 581773A21E37A2DB004E28A9 /* CoreMotion.framework */; };
-		581773AA1E37A2DB004E28A9 /* OpenAL.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 581773A31E37A2DB004E28A9 /* OpenAL.framework */; };
-		581773AB1E37A2DB004E28A9 /* OpenGLES.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 581773A41E37A2DB004E28A9 /* OpenGLES.framework */; };
-		581773AC1E37A2DB004E28A9 /* QuartzCore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 581773A51E37A2DB004E28A9 /* QuartzCore.framework */; };
-		581773AE1E37A7EF004E28A9 /* UIKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 581773AD1E37A7EF004E28A9 /* UIKit.framework */; };
-		C995F7171EAA5D0A003E34E1 /* assets in Resources */ = {isa = PBXBuildFile; fileRef = C995F7161EAA5D0A003E34E1 /* assets */; };
-		E21F0FA41F8922D8003761DD /* Media.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = E21F0FA31F8922D8003761DD /* Media.xcassets */; };
+    093EA0331288EEC18D3C79EC /* main.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 1E152A74253836079FA1075F /* main.cpp */; };
```

Comment:
Also, this nasty generated stuff looks like it might get regenerated using tabs... I'd have to check git blame.

Reply:
The questions is whether tabs or spaces were used originally. We can revert those files to pre-formatting commits.

Reply:
Reverted.